src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1395,7 +1400,12 @@ public XMPPError onSessionTerminate(JingleSession session, JingleIQ iq)
 
         synchronized (participantLock)
         {
-            terminateParticipant(participant, null, null, /* do not send session-terminate */ false);
+            terminateParticipant(
+                    participant,
+                    null,
+                    null,
+                    /* do not send session-terminate */ false,
+                    /* do send source-remove */ true);
```

Comment:
Can't we treat this case the same way as `onMemberLeft`?

Reply:
I thought I'd leave it, in case the participant sends a session-terminate, but does not leave the MUC. I don't know if thata happens in practice.